rolocController.cpp

```diff
@@ -447,12 +538,41 @@ void ROLOCcontroller::onFreqSetTimerExpired()
     mpFreqencySetTimer->stop();
 }
 
+/**
+ * @brief ROLOCcontroller::rolocSetDepthMode - set depth mode
+ * @param mode - mode enum
+ */
+void ROLOCcontroller::rolocSetDepthMode(ROLOC::eLINEFINDER_MODE mode)
+{
+    Q_UNUSED(mode);
+    qint16 depthPacket = 0;
+    depthPacket = 0x0200;
+    depthPacket |= mFrequency;
+    int status = m_i2cBus.i2c_writeWord(mI2cAddr, ROLOC::eCMD_INFO, depthPacket);
```

Comment:
simplify the equation. can you make this 0x0200 a constant ?

```suggestion
    qint16 depthPacket = 0x0200 | mFrequency ;
    int status = m_i2cBus.i2c_writeWord(mI2cAddr, ROLOC::eCMD_INFO, depthPacket);
```

Reply:
Yes